MyClass.java

```diff
@@ -54,15 +73,35 @@ public static void row(int spaces, int stars){
  for(i = 0; i < stars; i++){
               System.out.print("*");
           }
+<<<<<<< HEAD
+=======
           System.out.print("*");
           for(i = 0; i < stars; i++){
               System.out.print("*");
           }
+>>>>>>> bee68808de44320c3737a7c9146f8c7fa0194c59
           System.out.print("\n");
     }
     
     
   public static void stump(int width, int rows){ //Varibles can be changed; Need varible for width
+<<<<<<< HEAD
```

Comment:
I don't believe this is along with all the other lines like it E.G(=======) are executable,